src/main/java/org/folio/circulation/support/utils/ClockUtil.java

```diff
@@ -0,0 +1,213 @@
+package org.folio.circulation.support.utils;
+
+import static java.time.ZoneOffset.UTC;
+
+import java.time.Clock;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalDateTime;
+import org.joda.time.LocalTime;
+
+/**
+ * A clock manager for safely getting and setting the time.
+ */
+public class ClockUtil {
+  private static Clock clock = Clock.systemUTC();
```

Comment:
> I do not understand the reasoning behind why ClockManager is statically instatiation itself and then returning its instantiated self via its own static methods

That ClockManager instantiation is the singleton design pattern: https://www.journaldev.com/1377/java-singleton-design-pattern-best-practices-examples
It provides late instantiation, but late instantiation is not needed in mod-circulation, therefore your implementation using static methods is fine.